diff --git a/dist/lucide-vue-next.d.ts b/dist/lucide-vue-next.d.ts
index 31defc1dbb39037ee9b298d2ce0b8d8107e16072..ea8f94113bf44bcbbc3ec8e3b76dadbf41f3b30e 100644
--- a/dist/lucide-vue-next.d.ts
+++ b/dist/lucide-vue-next.d.ts
@@ -4,6 +4,7 @@ declare module 'lucide-vue-next'
 // Create interface extending SVGAttributes
 export interface SVGProps extends Partial<SVGAttributes> {
   size?: 24 | number
+  strokeWidth?: number
   absoluteStrokeWidth?: boolean
 }
 